src/components/BadDatePicker.vue

```diff
@@ -1,37 +1,30 @@
 <template>
-    <v-date-picker
-        @input="handleChange"
-        :min="min"
-        no-title
-        next-icon="mdi-menu-right"
-        prev-icon="mdi-menu-left"
-        color="primary"
-        elevation="1"
-        :full-width="fullWidth"
-        :value="value"
-        v-model="value"
-    ></v-date-picker>
+  <v-date-picker
```

Comment:
Is that change on the identation correct?

Reply:
Yes, it is correct. It is to avoid the warning with prettier when compiling and that change was suggested by Webstorm.